plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestHivePlans.java

```diff
@@ -310,11 +308,6 @@ public void testQueryScanningForTooManyPartitions()
                                                 exchange(REMOTE, REPLICATE,
                                                         project(
                                                                 tableScan("table_unpartitioned", Map.of("R_STR_COL", "str_col", "R_INT_COL", "int_col")))))))));
-
-        // The partitions will be loaded during split creation, so it fails during execution.
-        assertThatThrownBy(() -> getQueryRunner().execute(query))
-                .isInstanceOf(TrinoException.class)
-                .hasMessage("Query over table 'test_schema.table_int_with_too_many_partitions' can potentially read more than 5 partitions");
```

Comment:
Why removed? does it pass now?
do we have coverage for many partitions case and a join?

Reply:
Oops sorry I removed it accidentally.